qrenderdoc/Windows/PipelineState/GLPipelineStateViewer.ui

```diff
@@ -413,6 +422,9 @@
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolButton" name="meshView">
+            <property name="toolTip">
+             <string>View the mesh</string>
```

Comment:
This button exists in all the pipeline state viewers so it should have a tooltip in all of them or none.

Also apologies for the blatant nitpicking here, but the text doesn't quite read right to me. The window is called the "Mesh Output" although that's a pretty poor window really since that's not all it does. I think at least this should be:

> Open the Mesh Output view

or

> View the mesh input data